src/test/java/hudson/plugins/jira/JiraCreateReleaseNotesTest.java

```diff
@@ -65,9 +65,8 @@ public class JiraCreateReleaseNotesTest {
 
     @Before
     public void createCommonMocks() throws IOException, InterruptedException {
-        when(build.getProject()).thenReturn(project);
         when(build.getEnvironment(buildListener)).thenReturn(env);
-        when(buildListener.fatalError(Mockito.anyString(), Mockito.any())).thenReturn(printWriter);
+        when(buildListener.fatalError(Mockito.anyString(), Mockito.any(Object[].class))).thenReturn(printWriter);
```

Comment:
Adapting to the [new Mockito 5 behavior](https://github.com/mockito/mockito/releases/tag/v5.0.0) for varargs.